.gitignore

```diff
@@ -0,0 +1,8 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Use this as an example

https://github.com/github/gitignore/blob/master/Node.gitignore